discovery_engine_core/core/src/document.rs

```diff
@@ -99,8 +99,7 @@ pub enum UserReaction {
 }
 
 /// A d-dimensional sequence embedding.
-#[derive(Clone, Debug, Deref, Serialize, Deserialize)]
-#[cfg_attr(test, derive(Default))]
+#[derive(Clone, Debug, Deref, Serialize, Deserialize, Default)]
```

Comment:
It wasn't properly document, but the reason that we have the default only for test is because it does make sense to have a default implementation for this types outside tests.

Reply:
I expected as much, I just needed a way to write the conversion and wasn't clear on how to instantiate the types. Since we'll be dropping them, this is no longer an issue anyway.